### DISCUSSION TITLE
Error: A00001 Unreconciled syntax error

### DISCUSSION THREAD

**Discussion by trouble96** [xmos/xcore_iot#640] [Q&A]
I'm trying to build "example_freertos_getting_started" but 

[ 63%] Building C object CMakeFiles/tile1_example_freertos_getting_started.dir/modules/rtos/modules/FreeRTOS/FreeRTOS-SMP-Kernel/portable/ThirdParty/xClang/XCOREAI/port.c.obj
/tmp/ccsFkgaJ.s:6: Error: A00001 Unreconciled syntax error
make[3]: *** [CMakeFiles/tile1_example_freertos_getting_started.dir/build.make:485: CMakeFiles/tile1_example_freertos_getting_started.dir/modules/rtos/modules/FreeRTOS/FreeRTOS-SMP-Kernel/portable/ThirdParty/xClang/XCOREAI/port.c.obj] Error 1
make[2]: *** [CMakeFiles/Makefile2:3886: CMakeFiles/tile1_example_freertos_getting_started.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:3925: CMakeFiles/example_freertos_getting_started.dir/rule] Error 2
make: *** [Makefile:969: example_freertos_getting_started] Error 2

OS is Ubuntu 20.04.06 and xcc version is 15.3.0

**Comment by trouble96** [comment]
Also, trying to build under Windows 10 system (vs 2019), same error happened "A0001 Unconciled syntax error" in same file - /XCOREAI/port.c

**Comment by atomicdog** [comment]
I'm getting the same error.

**Comment by atomicdog** [comment]
Looks to be an issue with [15.3.0.](https://www.xcore.com/viewtopic.php?t=8944)
Use 15.2.1 instead.

**Reply by trouble96** [reply]
thank you very much!
